esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -257,6 +268,21 @@ void serialFlushRx(void)
 
 void setup_device_name(void)
 {
+#ifdef ESP8266
+    String MAC_buf;
+    MAC_buf = WiFi.macAddress();
+    // No point in any sort of hash here, the last bytes of the MAC address in hex is more useful
```

Comment:
I don't agree
pl do as for esp32
we want to keep it consistent and transparent

Reply:
The way it was done for the esp32 is far from transparent and not at all ideal, but I'll work on making the esp8266 use the same math since you insist.